php/libraries/TimePoint.class.inc

```diff
@@ -73,6 +73,22 @@ class TimePoint implements \LORIS\StudyEntities\AccessibleResource,
     ];
 
 
+    /**
+     * A DTO for TimePoint related data
+     */
+    protected ?TimePointData $data;
```

Comment:
I am not sure why this is kept as an instance variable instead of spread into 3 instance variables (sessionId, centerId and projectId)

Reply:
@ridz1208 @HenriRabalais any thoughts on inlining this into a TimePoint constructor rather than using a separate DTO object?

Reply:
It doesn't make a big different to me as long as we have a way to avoid the overhead of `singleton`/`select`.

Reply:
anything is better than what we have

Reply:
i like the idea of wraping contructors params in a type/class but I think those objects should be volatiles 🐦

Reply:
I don't think there is a volatile keyword indeed. My point is that what we need from that object is the values, not the structure. If we keep a reference to the object, I feel that we increase the coupling between the classes. I think we should take what we need and throw it away.

Reply:
I think a separate DTO object makes sense to me!